tests/integration/cartography/intel/aws/test_rds.py

```diff
@@ -143,3 +206,177 @@ def test_load_rds_snapshots_basic(neo4j_session):
         ),
     }
     assert actual_results == expected_results
+
+
+@patch.object(
+    cartography.intel.aws.rds,
+    "get_rds_snapshot_data",
+    return_value=DESCRIBE_DBSNAPSHOTS_RESPONSE["DBSnapshots"],
+)
+@patch.object(
+    cartography.intel.aws.rds,
+    "get_rds_instance_data",
+    return_value=DESCRIBE_DBINSTANCES_RESPONSE["DBInstances"],
+)
+@patch.object(
+    cartography.intel.aws.rds,
+    "get_rds_cluster_data",
+    return_value=DESCRIBE_DBCLUSTERS_RESPONSE["DBClusters"],
+)
+def test_sync_rds_comprehensive(
```

Comment:
This test covers the entire sync function instead of just the load with the previous basic tests